src/dumpsxiso/main.cpp

```diff
@@ -839,6 +870,7 @@ void ParseISO(cd::IsoReader& reader) {
 
     cd::ISO_DESCRIPTOR descriptor;
 	auto license = ReadLicense(reader);
+	const char* xa_edc = CheckEDCXA(reader);
```

Comment:
`tinyxml2::XMLElement::SetAttribute` taking a boolean exists, therefore you can modify `CheckEDCXA` to return `bool` and not a `"true"`/`"false"` string.